scanner/updater/export.go

```diff
@@ -183,6 +186,17 @@ func epssOpts() []updates.ManagerOption {
 	}
 }
 
+// TODO(ROX-26672): remove this.
+func redhatCSAFOpts() []updates.ManagerOption {
+	return []updates.ManagerOption{
+		// This is required to prevent default updaters from running.
+		updates.WithEnabled([]string{}),
+		updates.WithFactories(map[string]driver.UpdaterSetFactory{
+			"stackrox.rhel-csaf": csaf.NewFactory(),
```

Comment:
```suggestion
			"stackrox-rhel-csaf": csaf.NewFactory(),
```

Reply:
The name of the enricher is `stackrox.rhel-csaf` (named it that to try to confirm to the standard set in Claircore), and this map key must match the name of the enricher

Reply:
I think I may keep this as-is due to the conventions we have already set up in this file. We can consider a followup for this, though

Reply:
Looked into it a bit more, and I don't think that will always work, anyway. For example: we use `alpine`, but there is no `alpine` updater name. That's the name of the Factory which outputs several alpine-based updaters. I do not believe there is any variable called `alpine` we can use. That being said, we shouldn't let that affect every other updater/enricher. I can consider it for a followup PR

Reply:
Going to resolve this. This must match the name of the enricher, and the name has the `.` to follow the convention set by Claircore